Assets/Source/Components/Enemy/EnemyFormation.cs

```diff
@@ -0,0 +1,134 @@
+﻿using UnityEngine;
+using Assets.Source.Components.Base;
+using System.Collections.Generic;
+using Assets.Source.Constants;
+using System.Text;
+
+namespace Assets.Source.Components.Enemy
+{
+    public class EnemyFormation : ComponentBase
```

Comment:
This class works for now, but i think we will need to engineer this a lot further in order to abstractly use it, as well as use variations / patterns and what not (take a look at galaga's crazy ass movement patterns, etc)

Reply:
Yes I agree. This was a prototype.